docs/src/routes/examples/pages/basic/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<header>
+  <a href="#main-content" class="button focus-only skip-to-content">Skip to content</a>
+  <nav
+    data-open-label="Menu"
+    data-close-label="Sluit menu"
+    data-media="(min-width: 30rem)"
+    aria-label="Hoofdnavigatie"
+    class="collapsible"
+  >
+    <div class="collapsing-element">
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+</header>
+<main>
+  <section>
+    <div></div>
+  </section>
+</main>
+<footer>
+  <nav>
+    <h1>Manon Examples</h1>
```

Comment:
An `h1` in the footer is somewhat unexpected. Normally you'd expect the `h1` to be at the top of the `<main>`. Would it make sense to add an `h1` to main and change the footer heading to h2?